scripts/unix-install.sh

```diff
@@ -481,9 +481,13 @@ install_package()
 
   info "Setting permissions..."
   chmod +x "$agent_binary"
-  ln -sf "$agent_binary" "/usr/local/bin/$BINARY_NAME"
   succeeded
 
+  info "Linking binary $agent_binary => /usr/local/bin/$BINARY_NAME"
+  mkdir -p /usr/local/bin
```

Comment:
Will this create a `/usr/local/bin` with the wrong permissions? Does it matter? Looks like I have this folder on my M1 because of Docker and it's not writtable by me. I just want to make sure we don't screw up the users system by doing permissions wrong. 

This may not be a big concern though.

Reply:
A quick test shows that permissions are correct:

before
```
root@e755b29a5a3b:/# ls -la /usr/local/bin
total 8
drwxr-xr-x  2 root root 4096 Mar 16 17:07 .
drwxr-xr-x 10 root root 4096 Mar 16 17:07 ..
```

after
```
root@ad4813a7bdc8:/# rm -rf /usr/local/bin
root@ad4813a7bdc8:/# mkdir -p /usr/local/bin
root@ad4813a7bdc8:/# ls -la /usr/local/bin
total 8
drwxr-xr-x 2 root root 4096 Mar 29 00:16 .
drwxr-xr-x 1 root root 4096 Mar 29 00:16 ..
```

Reply:
Thanks @jsirianni